src/scss/_checkbox.scss

```diff
@@ -0,0 +1,124 @@
+/// @access private
+/// @param {Boolean} $additional-content Whether to output additional content styling
+/// @param {Boolean} $disabled Whether to output disabled state styling
+/// @param {Boolean} $inline-children Whether to output element inputs inline
+/// @param {Boolean} $inline-label Whether to output entire element inline with its label
+/// @param {Boolean} $invalid Whether to output invalid state styling
+/// @param {Boolean} $right Whether to display checkbox to the right of text
+/// @output Styling for text inputs
+@mixin _oFormsCheckbox(
+	$additional-content: null,
+	$disabled: null,
+	$inline-children: null,
+	$inline-label: null,
+	$invalid: null,
+	$right: null
+) {
+	.o-forms--checkbox {
+		label span {
```

Comment:
The `label` I'm fine with because there can only be one per input in valid HTML, but the `span` selector could be problematic if someone needs to add in any amount of custom stuff over the top of o-forms, or even if we later decide we need a new feature and a span happens to be the right element for that. Wonder whether we should adopt a class like `o-forms__label-text`?

(this is in more places, just commented on the first one I found)

Reply:
I would argue that element specificity is low enough for a new class to be applied to a span and take precedence? 🤔

Reply:
I concur with Rowan's feedback 👍 Relying on users to override styles we apply (and may change in minor or patch releases) is unreliable.